modules/soc_psa/main.sh

```diff
@@ -100,7 +102,7 @@ if (($soccalc == 0)); then #manual calculation not enabled, using existing logic
 		incrementTimer
 	else
 		echo 0 > $soctimerfile
-		sudo python $MODULEDIR/psasoc.py $CHARGEPOINT $username $password $clientId $clientSecret $manufacturer $soccalc
+		sudo python $MODULEDIR/psasoc.py $CHARGEPOINT $username $password $clientId $clientSecret $manufacturer $soccalc $vehicleId
```

Comment:
```suggestion
		sudo python "$MODULEDIR/psasoc.py" "$CHARGEPOINT" "$username" "$password" "$clientId" "$clientSecret" "$manufacturer" "$soccalc" "$vehicleId"
```
Andernfalls führt Bash eine string-expansion auf jeden Parameter durch. Wenn in einem Parameter ein Stern oder ein Leerzeichen oder sowas vorkommt, dann führt das ganze zu Überraschungen. Zumindest mal für das Passwort erscheint mir das realistisch, aber auch für alles andere schaden die Anführungszeichen nicht.

Das gleiche gilt auch für die Pythonaufrufe aus den anderen Bash-Dateien.

Reply:
Das kann ich gerne noch ändern.

Reply:
@benderl Done